module1.js

```diff
@@ -81,6 +97,9 @@ function question5(number1, number2) {
 
 function question6(array) {
   // TODO:
+  var longitud = array.length;
```

Comment:
This isn't really relevant - just want to point out that "longitude" probably wasn't what you meant here :D